etn-ccis/blui-colors
# Update build of Color package to allow nested import

Fixes 4939 .



#### Changes proposed in this Pull Request:

- Allow nested import for branding
- 
-



#### Screenshots / Screen Recording (if applicable)

- 
<img width="621" alt="image-20231116-144449" src="https://github.com/etn-ccis/blui-colors/assets/133877691/9893efed-a06f-48eb-880e-2207a40d189d">



#### To Test:

- Run `yarn build`
- Copy branding and dist folder in RN showcase demo to check colors.
- Use the above image for reference to import 
- Update readme



#### Any specific feedback you are looking for?

-
